src/js/parsers/mediaparser.js

```diff
@@ -83,7 +84,6 @@ const mediaparser = function (obj, item) {
                     mediaparser(node, item);
                     break;
                 case 'subtitle':
```

Comment:
Wrap the following statements in a block instead of defining entry further up in scope.